src/components/Calculator.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+
+export default class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <>
+        <div className="container">
+          <div className="display">
+            <span>0</span>
+          </div>
+          <div className="row">
+            <button type="button" className="btn">AC</button>
+            <button type="button" className="btn">+/-</button>
+            <button type="button" className="btn">%</button>
+            <button type="button" className="btn operator">÷</button>
+          </div>
+          <div className="row">
+            <button type="button" className="btn">7</button>
+            <button type="button" className="btn">8</button>
+            <button type="button" className="btn">9</button>
+            <button type="button" className="btn operator">x</button>
+          </div>
+          <div className="row">
+            <button type="button" className="btn">4</button>
+            <button type="button" className="btn">5</button>
+            <button type="button" className="btn">6</button>
+            <button type="button" className="btn operator">-</button>
+          </div>
+          <div className="row">
+            <button type="button" className="btn">1</button>
+            <button type="button" className="btn">2</button>
+            <button type="button" className="btn">3</button>
+            <button type="button" className="btn operator">+</button>
+          </div>
+          <div className="last-row">
+            <button type="button" className="btn">0</button>
+            <button type="button" className="btn">.</button>
+            <button type="button" className="btn operator">=</button>
+          </div>
+        </div>
+      </>
+    );
+  }
+}
```

Comment:
- [ ] **README.md**: As required for every **[project done at Microverse](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)** kind make sure you add a descriptive README file with all main headers present from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) regarding how much into the details they go in each section of the README.